### PR TITLE
1546 duration use

### PR DESCRIPTION
**Pull request by JNATION85**
Durations example uses min and max attributes which are invalid for text inputs

### CODE REVIEW

**Review by jrbarnes9 — APPROVED** [review]

**Review by jrbarnes9 — CHANGES_REQUESTED** [review]
Also needs removing from the mutually exclusive duration example: https://ons-design-system.netlify.app/components/mutually-exclusive/examples/duration/

**Review by JNATION85 — COMMENTED** [review]
Pushed changes

File: src/components/mutually-exclusive/examples/duration/index.njk [hunk 1]
```diff
@@ -11,21 +11,11 @@
         "dontWrap": true,
         "field1": {
             "id": "address-duration-years",
-            "name": "address-duration-years",
-            "label": "Years",
-            "attributes": {
-                "min": 0,
-                "max": 100
-            }
+            "name": "address-duration-years"
```
Comment (rmccar) [inline src/components/mutually-exclusive/examples/duration/index.njk:14]:
```suggestion
            "name": "address-duration-years",
            "label": "Years"
```

File: src/components/mutually-exclusive/examples/duration/index.njk [hunk 2]
```diff
@@ -11,21 +11,11 @@
...
         },
         "field2": {
             "id": "address-duration-months",
-            "name": "address-duration-months",
-            "label": "Months",
-            "attributes": {
-                "min": 0,
-                "max": 11
-            }
+            "name": "address-duration-months"
```
Comment (rmccar) [inline src/components/mutually-exclusive/examples/duration/index.njk:18]:
```suggestion
            "name": "address-duration-months",
            "label": "Months"
```

Comment (rmccar) [reply src/components/mutually-exclusive/examples/duration/index.njk:18]: These two labels need to be added back in. Other than that looks good!

**Review by jrbarnes9 — APPROVED** [review]

**Comment by rmccar** [comment]
[Issue #1546: Durations example uses `min` and `max` attributes which are invalid for `text` inputs](https://trello.com/c/G8oNTmj8/1194-issue-1546-durations-example-uses-min-and-max-attributes-which-are-invalid-for-text-inputs)

**Pull request merged**